content/en/projects/_index.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: "Projects"
+description: "ACL Special Interest Group on Turkic Languages Projects"
+date: 2020-10-06T08:49:55+00:00
+lastmod: 2020-10-06T08:49:55+00:00
```

Comment:
I'm not really familiar with where this metadata is used, but should it say 2022?

Reply:
I will remove these, leftover